helpers/compile/plugins/fill-plugin/fillers/fs.ts

```diff
@@ -2,13 +2,14 @@ export function existsSync() {
   return false
 }
 
+export const promises = {}
```

Comment:
Needed for esbuild although code requiring this gets tree-shaken. Not an issue, but strange.